internal/local/command_kafka_start.go

```diff
@@ -78,6 +78,7 @@ func (c *command) kafkaStart(cmd *cobra.Command, _ []string) error {
 
 	containers, err := dockerClient.ContainerList(context.Background(), types.ContainerListOptions{All: true})
 	if err != nil {
+		output.ErrPrintln(false, "The local commands have been verified to work with Docker Desktop version as late as 4.25.0.")
```

Comment:
We should probably make this a string so that we don't have to modify all instances of it whenever the newest verified version needs changing.